slashing-protection/src/main/resources/migrations/postgresql/V1__initial.sql

```diff
@@ -6,15 +6,15 @@ CREATE TABLE validators (
 CREATE TABLE signed_blocks (
     validator_id INTEGER NOT NULL,
     slot NUMERIC(20) NOT NULL,
-    signing_root BYTEA,
+    signing_root BYTEA NOT NULL,
```

Comment:
We shouldn't change what V1 was as there will be existing databases using this schema

Reply:
deal